app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+    belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+    belongs_to :post
+
+    after_create :update_comment_counter
+
+    def update_comment_counter
+      post.update(comments_counter: post.comments.count)
+    end
```

Comment:
- [ Optional ] Kindly I recommend you to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #`update_comment_counter `method every time a new post has been saved to database, then make the `update_comment_counter` method private.

 ```
after_save : update_comment_counter

  private

  def update_comment_counter
      post.update(comments_counter: post.comments.count)
    end
```